stylesheets/_component.flash.scss

```diff
@@ -111,8 +111,13 @@
 }
 
 .flash-container {
-    left: $nav-primary-width;
+    left: 0;
```

Comment:
`flash-container` is no longer absolutely positioned so this change should not be needed